CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format is based on [Common Changelog](https://common-changelog.org).\
 Unlike Common Changelog, the `unreleased` section of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) is preserved with the addition of a tag to specify which type of release should be published and to foster discussions about it inside pull requests. This tag should be one of the names mandated by SemVer, within brackets: `[patch]`, `[minor]` or `[major]`. For example: `## Unreleased [minor]`.
 
-## Unreleased
+## Unreleased [minor]
+
+### Changed
+- **Breaking:** Change exposed CLI commands
```

Comment:
Clarify the changes: the reader should know easily how to update their code when they upgrade the lib 🙂 
```suggestion
- **Breaking:** Change exposed CLI commands:
  - `ota-track` becomes `ota track`. 
```

…